lib/preview-web/src/PreviewWeb.tsx

```diff
@@ -398,6 +399,10 @@ export class PreviewWeb<TFramework extends AnyFramework> {
     await this.onUpdateArgs({ storyId, updatedArgs });
   }
 
+  async onPreloadStory(ids: string[]) {
```

Comment:
If it's an array of ids, should we call this `preloadStories`?

Reply:
done!